docs/gettingstarted/index.md

```diff
@@ -6,18 +6,19 @@ This is a simple guide to get you from nothing to a basic mod. The rest of this
 From Zero to Modding
 --------------------
 
-1. Obtain the Mod Development Kit (MDK) from forge's [files][] site. (Look for the MDK file type)
-2. Extract the downloaded MDK to an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
+1. Obtain a Java 8 Development Kit (JDK) and a 64-bit Java Virtual Machine (JVM). Minecraft and MinecraftForge both compile against Java 8 and as such should be used for development. Using a 32-bit JVM will result in some problems when running the below gradle tasks. You can obtain one from [AdoptOpenJDK][jdk].
```

Comment:
Would there be merit in having the part about a 64-bit vs 32-bit JVM be in an admonition instead of directly inline?
I would assume the vast majority of computers today (or at least, those used by people) are 64-bit, and that the AdoptOpenJDK shows the 64-bit variant of the JDK if it detects the user is on a 64-bit computer.

Reply:
Since there hasn't been any development or major releases of computers with 32-bit systems, the admonishment probably wouldn't make much of a difference. We could put the first point it in a `!!!important` tag instead to indicate the importance of using it, but I don't think it would have a major effect. I don't think I've ever seen on forgecord anyone mention using a 32-bit system to mod, but I've also only been around since September.